pkg/operations/cordondrainvm.go

```diff
@@ -62,7 +62,16 @@ func SafelyDrainNode(az armhelpers.ACSEngineClient, logger *log.Entry, masterURL
 
 	//Evict pods in node
 	drainOp := &drainOperation{client: client, node: node, logger: logger, timeout: timeout}
-	return drainOp.deleteOrEvictPodsSimple()
+	if err = drainOp.deleteOrEvictPodsSimple(); err != nil {
```

Comment:
The original usage of `return drainOp.deleteOrEvictPodsSimple()` is preferred here.